mteb/tasks/Retrieval/hun/HunSum2.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from datasets import load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class HunSum2AbstractiveRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="HunSum2AbstractiveRetrieval",
+        dataset={
+            "path": "SZTAKI-HLT/HunSum-2-abstractive",
+            "revision": "24e1445c8180d937f0a16f8ae8a62e77cc952e56",
+        },
+        description=(
+            "HunSum-2-abstractive is a Hungarian dataset containing news articles along with lead, titles and metadata."
+        ),
+        reference="https://arxiv.org/abs/2404.03555",
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["hun-Latn"],
+        main_score="ndcg_at_10",
```

Comment:
```suggestion
        main_score="ndcg_at_1",
```
Given the very high scores I would lower it to 1.

Reply:
Okay will change